references/proxy-template/pipeline.sh

```diff
@@ -18,17 +18,37 @@ set -x
 
 SCRIPTPATH="$( cd "$(dirname "$0")" || exit >/dev/null 2>&1 ; pwd -P )"
 
+# default proxy name and version
+PROXY=example 
+VERSION=v1
+
 # clean up
-rm -rf example-v1
+rm -rf "$PROXY"-"$VERSION"
 
 # deploy all shared flows
 (cd "$SCRIPTPATH"/../common-shared-flows && sh deploy.sh all --apigeeapi)
 
+# set target server environment variables
+DEFAULT_TARGET_URL=https://httpbin.org/get
+export TARGET_URL="${TARGET_URL:-"$DEFAULT_TARGET_URL"}"
+
+# shellcheck disable=SC1091
+. ./set-targetserver-envs.sh
```

Comment:
why is disabling SC1091 needed here? Can't you use the SCRIPTPATH instead?

Reply:
SCRIPTPATH added